services/web-app/components/divider/divider.module.scss

```
@@ -0,0 +1,20 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '@carbon/react/scss/breakpoint' as breakpoint;
+@use '@carbon/react/scss/spacing' as spacing;
+@use '@carbon/react/scss/theme' as theme;
+@use '@carbon/react/scss/type' as type;
+
+.divider {
+  padding: spacing.$spacing-07 0 calc(spacing.$spacing-09 + spacing.$spacing-03);
+  border-top: 1px solid theme.$border-strong;
```

```suggestion
  padding-top: spacing.$spacing-07;
  border-top: 1px solid theme.$border-strong;
  margin-top: calc(spacing.$spacing-09 + spacing.$spacing-03);
```

suggestion to get correct spacing above the first element